src/components/DashboardPeriodSelector.vue

```diff
@@ -119,6 +122,7 @@ export default {
   },
 
   methods: {
+    ...mapActions(['setSelectedPeriod']),
```

Comment:
Litt usikker på om det blir rett å sette perioden i store her? Disse lytter vel i utgangspunktet på endringer "utenfra", så burde `setSelectedPeriod` ev. erstatte `setPeriod` i `DashboardResultIndicatorsSection.vue`, og `this.currentResultIndicatorPeriod` erstattes av `selectedPeriod` state fra store?

Ett problem er at `selectedPeriod` aldri blir satt dersom man f.eks. ikke har noen målinger av type "Resultatindikator". I så fall kastes en del feil. Periode-velgeren blir heller ikke tilgjengelig.

Bør vi på et eller annet vis ha en default-periode (?) som settes i router-guarden for målinger/dashbordet? 🤔

Reply:
Kanskje vi må løsrive periodevelgeren først? 😅